src/lsg/patterns/element/index.tsx

```diff
@@ -113,11 +114,11 @@ const StyledElementLabel = styled.div`
 
 const StyledPlaceholder = styled.div`
```

Comment:
@LKuechler Is it possible to display the Placeholder ONLY when an element is being dragged around?